src/order-history/service.js

```diff
@@ -1,23 +1,72 @@
 import { getAuthenticatedHttpClient, getAuthenticatedUser } from '@edx/frontend-platform/auth';
-import { getConfig } from '@edx/frontend-platform';
+import { getConfig, initialize, mergeConfig } from '@edx/frontend-platform';
 
 const { ECOMMERCE_BASE_URL } = getConfig();
+
+// Temporary: until we add COMMERCE_COORDINATOR_BASE_URL in frontend-platform, use mergeConfig to join it
+// with the rest of the config items (so we don't need to get it separately from process.env). After we add
+// it to frontend-platform, we'll be able to set it like ECOMMERCE_BASE_URL above, and block below can go away.
+initialize({
+  handlers: {
+    config: () => {
+      mergeConfig({
+        COMMERCE_COORDINATOR_BASE_URL: process.env.COMMERCE_COORDINATOR_BASE_URL || null,
+      });
+    },
+  },
+});
+
 const ECOMMERCE_API_BASE_URL = `${ECOMMERCE_BASE_URL}/api/v2`;
 const ECOMMERCE_RECEIPT_BASE_URL = `${ECOMMERCE_BASE_URL}/checkout/receipt/`;
 
 // eslint-disable-next-line import/prefer-default-export
 export async function getOrders(page = 1, pageSize = 20) {
   const httpClient = getAuthenticatedHttpClient();
   const { username } = getAuthenticatedUser();
+  const { COMMERCE_COORDINATOR_BASE_URL } = getConfig();
+  const orderFetchingUrl = `${COMMERCE_COORDINATOR_BASE_URL}/orders/order_history/`;
 
-  const { data } = await httpClient.get(`${ECOMMERCE_API_BASE_URL}/orders/`, {
+  // [START] TEMPORARY CODE for rollout testing/confirmation===========
+  // Call ecommerce for order info including waffle flag
+  let { data } = await httpClient.get(`${ECOMMERCE_API_BASE_URL}/orders/`, {
     params: {
       username,
       page,
       page_size: pageSize,
     },
   });
 
+  let callCC = false;
+  if (data.count > 0) {
+    callCC = data.results[0].enable_hoist_order_history;
+  }
+  console.log('DKTEMP: enable_hoist_order_history flag is: ', callCC);
+
+  if (callCC) {
+    console.log('DKTEMP: about to call commerce-coordinator');
+    const newData = await httpClient.get(orderFetchingUrl, {
+      params: {
+        username,
+        page,
+        page_size: pageSize,
+      },
+    });
+    data = newData.data;
```

Comment:
Yes these will get the same json results, the only difference being that one will go through commerce-coordinator. I do the replacement because even though these results aren't actually different, when the waffle flag is enabled we want the order history page to show the data coming back from commerce-coordinator. (As opposed to just calling the commerce-coordinator in the background but not actually showing the data we get from it).